tests/app/forms/field_handlers/conftest.py

```diff
@@ -2,45 +2,142 @@
 
 from app.data_models.answer_store import AnswerStore
 from app.data_models.list_store import ListStore
+from app.forms.field_handlers.select_handlers import Choice, ChoiceWithDetailAnswer
 from app.questionnaire import QuestionnaireSchema
+from app.questionnaire.rules.rule_evaluator import RuleEvaluator
 from app.questionnaire.value_source_resolver import ValueSourceResolver
 
 
-@pytest.fixture
-def value_source_resolver():
-    schema = QuestionnaireSchema(
+def get_mock_schema():
+    return QuestionnaireSchema(
         {
             "questionnaire_flow": {
                 "type": "Linear",
                 "options": {"summary": {"collapsible": False}},
             }
         }
     )
+
+
+def get_mock_answer_store():
+    return AnswerStore()
+
+
+def get_mock_list_store():
+    return ListStore()
+
+
+def get_mock_metadata():
+    return {}
+
+
+def get_mock_response_metadata():
+    return {"started_at": "2021-01-01T09:00:00.220038+00:00"}
+
+
+def get_mock_location():
+    return None
+
+
+def get_mock_list_item_id():
+    return None
```

Comment:
Slightly odd, would probably have just done this inline. Or at least make the mock more representative, i.e get_mock_location_none.

Reply:
Was to ensure the rule evaluator and value source resolver stayed in sync, but have now removed these and made it inline for all those that return a simple default value.